### PR TITLE
feat: error messages proxy when run without args

### PR DESCRIPTION
**Pull request by LiamThys**

### CODE REVIEW

**Review by BelgianNoise — DISMISSED** [review]
Works fine for me

File: packages/dgt-id-proxy/lib/main.spec.ts [hunk 1]
```diff
@@ -122,39 +122,6 @@ describe('Main.ts', () => {
...
-
-    });
-
-    it('should return the default openidConfigurationFilePath & jwksFilePath if none was given', () => {
-
-      expect(createVariables([ 'npm run start', '--', '-c', configPath, '-U', 'https://digita-ai.eu.auth0.com/', '-i',  'fx0Ujpt78FfdsFdgfs8fssAW3QWF', '-s', 'SdfvZh7egyafNcwnj33KFy1xdsffFD8fdSW8xcDF8to', '-A', 'https:​//useid.eu.auth0.com/api/v2/' ]))
-        .toEqual(variables);
-
-    });
-
```
Comment (lem-onade) [inline packages/dgt-id-proxy/lib/main.spec.ts:157]: I'm not sure these test should have been deleted @AbelVandenBriel, coverage was also lowered.

**Review by lem-onade — CHANGES_REQUESTED** [review]

File: packages/dgt-id-proxy/lib/main.spec.ts (hunk 1 above)
Comment (BelgianNoise) [reply packages/dgt-id-proxy/lib/main.spec.ts]: I added the first test back. Left out the second one as it is not relevant anymore since this PR.

**Review by AbelVandenBriel — APPROVED** [review]

**Review by AbelVandenBriel — APPROVED** [review]

**Review by JasperVandenberghen — APPROVED** [review]
Works fine.

One question: is it automatically translated to my Mac's language settings? Cause if not I don't see why it should contain dutch.

image

**Comment by lem-onade** [comment]
> One question: is it automatically translated to my Mac's language settings? Cause if not I don't see why it should contain dutch.

Yes, seems to use system language:
```
➜  dgt-id-proxy git:(feat/error-message-proxy-without-args) npm run start

> @digita-ai/dgt-id-proxy@0.19.9 start /Users/stijn/Documents/digita/dgt-id-broker/packages/dgt-id-proxy
> node ./bin/server.js

node ./dist/main.js [args]

Options:
      --version                      Show version number               [boolean]
  -c, --config                                                          [string]
  -C, --urlWebIdFactoryClaim                                            [string]
  -u, --proxyUri                     URL of the proxy        [string] [required]
  -U, --upstreamUri                  URL of the upstream server
                                                             [string] [required]
  -m, --mainModulePath                                                  [string]
  -o, --openidConfigurationFilePath  relative path to the OIDC configuration
                                                             [string] [required]
  -j, --jwksFilePath                 relative path to the JWKs
                                                             [string] [required]
  -r, --redirectUri                                                     [string]
  -i, --clientId                                                        [string]
  -s, --clientSecret                                                    [string]
  -P, --proxyTokenUrl                                                   [string]
  -L, --proxyClientUrl                                                  [string]
  -A, --auth0Api                                                        [string]
      --help                         Show help                         [boolean]

Missing required arguments: proxyUri, upstreamUri, openidConfigurationFilePath, jwksFilePath
```

is my output

**Review by lem-onade — APPROVED** [review]

**Comment by BelgianNoise** [comment]
> One question: is it automatically translated to my Mac's language settings? Cause if not I don't see why it should contain dutch.

Guess so. Mine was in english.

**Pull request merged**